### ISSUE TITLE
Make sure sidebar is hidden from assistive technology when map/embed is showing

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#931]
**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/3-a-1/

When the maps or dashboard tabs are opened, the ‘Sub-categories’ toolbar visually disappears, but are not hidden from assistive technology. Screen reading software still relays these.

**Solution:**
Please ensure content which should be hidden is done so in a way that is hidden from assistive technology. This can be achieved with the CSS style ‘display: none;’.
